falcon/testing/base.py

```diff
@@ -1,123 +0,0 @@
-# Copyright 2013 by Rackspace Hosting, Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-"""Test case base class (deprecated)."""
-
-import itertools
-
-try:
-    import testtools as unittest
-except ImportError:  # pragma: nocover
-    import unittest
-
-import falcon
-import falcon.request
-from falcon.testing.helpers import create_environ
-from falcon.testing.srmock import StartResponseMock
-
-
-# NOTE(kgriffs): Since this class is deprecated and we will be using it
-# less and less for Falcon's own tests, coverage may be reduced, hence
-# the pragma to ignore coverage errors from now on.
-class TestBase(unittest.TestCase):  # pragma nocover
```

Comment:
We have to keep this around for now. While it's deprecated, we can't remove this until 2.0.